src/web/src/components/Posts/Post.tsx

```diff
@@ -311,62 +353,90 @@ const PostComponent = ({ postUrl, currentPost, totalPosts }: Props) => {
-              <h1 className={classes.author}>
-                <a className={classes.link} href={post.feed.link}>
-                  {post.feed.author}
-                </a>
-              </h1>
-            </div>
-            <div className={classes.publishedDateContainer}>
-              <h1 className={classes.published}>
-                <a href={post.url} rel="bookmark" className={classes.link}>
-                  <time dateTime={post.updated}>{`${formatPublishedDate(post.updated)}`}</time>
-                </a>
+      {!desktop && (
+        <>
+          <Accordion className={classes.accordion}>
+            <AccordionSummary className={classes.accordionSummary}>
+              <ListSubheader component="div" className={classes.postInfo}>
+                <div className={classes.titleContainer}>
+                  <Typography
+                    variant="h3"
+                    title={post.title}
+                    id={post.id}
+                    className={clsx(classes.title, expandHeader && classes.expandedTitle)}
+                  >
+                    <span
+                      role="button"
+                      tabIndex={0}
+                      onClick={() => setExpandHeader(!expandHeader)}
+                      onKeyDown={() => setExpandHeader(!expandHeader)}
+                    >
+                      {post.title}
+                    </span>
+                  </Typography>
+                </div>
+                <div className={classes.authorAvatarContainer}>
+                  <PostAvatar name={post.feed.author} url={post.feed.link} />
+                </div>
+                <div className={classes.authorNameContainer}>
+                  <h1 className={classes.author}>
+                    <a className={classes.link} href={post.feed.link}>
+                      {post.feed.author}
+                    </a>
+                  </h1>
+                </div>
+                <div className={classes.publishedDateContainer}>
+                  <h1 className={classes.published}>
+                    <a href={post.url} rel="bookmark" className={classes.link}>
+                      <time dateTime={post.updated}>{`${formatPublishedDate(post.updated)}`}</time>
+                    </a>
 
-                <ShareButton url={post.url} />
-              </h1>
+                    <ShareButton url={post.url} />
+                  </h1>
 
-              <div>
-                <AdminButtons />
-              </div>
-            </div>
-          </>
-        )}
-      </ListSubheader>
+                  <div>
+                    <AdminButtons />
+                  </div>
+                </div>
+              </ListSubheader>
+            </AccordionSummary>
+            <AccordionDetails>
+              {!!extractedGitHubUrls.length && <GitHubInfoMobile ghUrls={extractedGitHubUrls} />}
+            </AccordionDetails>
+          </Accordion>
+        </>
+      )}
       {desktop && (
```

Comment:
It'll be nice if y can combine these two expression into something like `desktop ? <A/> : <B/>`